cmd/agent/entrypoint.go

```diff
@@ -41,9 +40,9 @@ type Entrypoint struct {
 
 	srv          *server.Server
 	promMetrics  *metrics.Agent
-	lokiLogs     *loki.Logs
+	lokiLogs     *logs.Logs
```

Comment:
My editor was complaining about this; we had the package imported twice with a different alias.